src/Symfony/Component/Validator/Constraints/PasswordStrengthValidator.php

```diff
@@ -34,43 +36,37 @@ public function validate(#[\SensitiveParameter] mixed $value, Constraint $constr
         if (!\is_string($value)) {
             throw new UnexpectedValueException($value, 'string');
         }
+        $passwordStrengthEstimator = $this->passwordStrengthEstimator ?? [$this, 'estimate'];
```

Comment:
first-class callable + renaming of the method?
`$passwordStrengthEstimator = $this->passwordStrengthEstimator ?? $this->estimateStrength()`